charts/admission-controller/Chart.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: v2
 name: admission-controller
 description: Sysdig Admission Controller using Sysdig Secure inline image scanner
 type: application
-version: 0.14.14
+version: 0.14.15
 appVersion: 3.9.35
 home: https://sysdiglabs.github.io/admission-controller/
 icon: https://avatars.githubusercontent.com/u/5068817?s=200&v=4
 maintainers:
-  - name: giuse-sysdig
-    email: giuseppe.esposito@sysdig.com
+  - name: guidobonomi
```

Comment:
this is not correct: Admission controller is now under platform team and is not in VM. Maybe @acaprari can help identifying the proper ownership for this